addon/components/frost-popover.js

```diff
@@ -142,6 +143,21 @@ export default Component.extend(PropTypeMixin, {
 
       $(target).on(event, this._eventHandler)
     }
+
+    // handle mouse events on visible popover
+    $(popover).on('mouseenter', event => {
```

Comment:
Might be a good idea to use `off` when the component dies. Even if `popover` is destroyed when the component dies, the event listeners are still attached due to how Ember destroys the DOM.

Reply:
Umm... yes. A really good idea. Thank you.